src/components/Details.js

```diff
@@ -0,0 +1,99 @@
+              <p className="dat-type">SOURCE:</p>
+              <b className="dat-type">{cou.data.source}</b>
+            </div>
+            <div className="sor">
+              <p className="dat-type">TOTAL CONFIRMED:</p>
+              <b className="dat-type">{cou.data.today_confirmed}</b>
+            </div>
+            <div className="sor">
+              <p className="dat-type">TOTAL DEATH:</p>
+              <b className="dat-type">{cou.data.today_deaths}</b>
+            </div>
+            <div className="sor">
+              <p className="dat-type">CONFIRMED:</p>
+              <b className="dat-type">{cou.data.today_new_confirmed}</b>
+            </div>
+            <div className="sor">
+              <p>TODAY DEATH:</p>
+              <b className="dat-type">{cou.data.today_new_deaths}</b>
+            </div>
+            <div className="sor">
+              <p>Today New Open Cases:</p>
+              <b className="dat-type">{cou.data.today_new_open_cases}</b>
+            </div>
+            <div className="sor">
+              <p>Today New Recoveries:</p>
+              <b className="dat-type">{cou.data.today_new_recovered}</b>
+            </div>
+            <div className="sor">
+              <p>Today Open Cases:</p>
+              <b className="dat-type">{cou.data.today_open_cases}</b>
+            </div>
+            <div className="sor">
+              <p>Today Recovered:</p>
+              <b className="dat-type">{cou.data.today_recovered}</b>
+            </div>
+            <div className="sor">
+              <p>Today vs Yesterday Confirmed:</p>
+              <b className="dat-type">{cou.data.today_vs_yesterday_confirmed}</b>
+            </div>
+            <div className="sor">
+              <p>Today vs Yesterday Deaths:</p>
+              <b className="dat-type">{cou.data.today_vs_yesterday_deaths}</b>
+            </div>
+            <div className="sor">
+              <p>Today vs Yesterday Open Cases:</p>
+              <b className="dat-type">{cou.data.today_vs_yesterday_open_cases}</b>
+            </div>
+            <div className="sor">
+              <p>Today vs Yesterday Recovered:</p>
+              <b className="dat-type">{cou.data.today_vs_yesterday_recovered}</b>
+            </div>
+            <div className="sor">
+              <p>Yesterday Confirmed:</p>
+              <b className="dat-type">{cou.data.yesterday_confirmed}</b>
+            </div>
+            <div className="sor">
+              <p>Yesterday Deaths:</p>
+              <b className="dat-type">{cou.data.yesterday_deaths}</b>
+            </div>
+            <div className="sor">
+              <p>Yesterday Open Cases:</p>
+              <b className="dat-type">{cou.data.yesterday_open_cases}</b>
+            </div>
+            <div className="sor">
+              <p>Yesterday Recovered:</p>
+              <b className="dat-type">{cou.data.yesterday_recovered}</b>
+            </div>
+          </div>
```

Comment:
Currently, I am unable to check each countries details with your deployed webpage. It is required that the project is deployed properly, the home and detail page works as expected.

![image](https://user-images.githubusercontent.com/66112760/152362071-50697586-6343-4e46-ae69-085042964eb4.png)

Kindly fix this 👍